README.md

```diff
@@ -109,7 +121,12 @@ Verification (rather than stub it) all non-relative urls will cause the handover
    "deskProServiceId" : "abc",
    "signOutUrl" : "/sign-out",
    "accessibilityUrl" : "/accessibility-statement/my-service",
-   "regime" : "VATC"
+   "regime" : "VATC",
+   "labels" : {
+                 "cy" : {
+                   "optServiceName" : "Service name translated into welsh"
+                   }
+               }
```

Comment:
a lot of extra spaces, can we apply indentation please?

Reply:
sure